students/11th/hyojeung/style/common.css

```
@@ -0,0 +1,11 @@
+*{
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    box-sizing: border-box;
+  }
+
+*:focus {outline:none;}
```

```suggestion
*:focus {
  outline:none;
}
```
한줄 보다는 펼쳐서 쓰는게 좋겠습니다. 나머지 요소들도 마찬가지입니다.

---

펼쳐서 쓰는 방향으로 모두 수정하였습니다 👍  동일하지 않았던 줄바꿈도 변경하였습니다.